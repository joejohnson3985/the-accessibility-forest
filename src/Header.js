import React, { Component } from 'react';
import './Header.scss';

class Header extends Component {
  constructor() {
    super();
    this.state = {

    }
  }

  render() {
    return (
      <header className="header-container">
        <hgroup className="header-text"> 
          <h1>The Accessibility Forest</h1>
          <h3>Learn about the accessibility tree to grow your forest!</h3>
        </hgroup>
      </header>

    );
  }
}

export default Header;