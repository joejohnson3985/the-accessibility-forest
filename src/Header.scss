@import './_variables.scss';

.header-container {
  display: flex;
  align-items: center;
  justify-content: space-between;
  background-color: $accent-color1;
}

.header-text {
  color: $primary-color;
  margin-left: 40px;
}

.header-nav {
  display: flex;
  align-items: center;
  justify-content: space-between;
  margin-right: 40px;
  width: 500px;
}

.header-nav div {
  @include primary-button;

  &:hover {
    background-color: $secondary-color-hover;
  }
}
