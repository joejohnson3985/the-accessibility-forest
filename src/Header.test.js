import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme';

describe('Header', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
        <Header />
      )
  });

   it('Should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
})