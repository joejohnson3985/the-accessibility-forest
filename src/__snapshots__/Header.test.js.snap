// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should match the snapshot 1`] = `
<header
  className="header-container"
>
  <hgroup
    className="header-text"
  >
    <h1>
      The Accessibility Forest
    </h1>
    <h3>
      Learn about the accessibility tree to grow your forest!
    </h3>
  </hgroup>
</header>
`;
